<?xml version="1.0" ?>
<!-- =================================================================================== -->
<!-- |    This document was autogenerated by xacro from /home/gjohn/ros_ws/src/chargepal_description/urdf/environment/primitive_adapter_station.urdf.xacro | -->
<!-- |    EDITING THIS FILE BY HAND IS NOT RECOMMENDED                                 | -->
<!-- =================================================================================== -->
<robot name="primitive_adapter_station">
  <!-- start kinematic chain -->
  <link name="world"/>
  <joint name="world_joint" type="fixed">
    <parent link="world"/>
    <child link="low_body_link"/>
    <origin rpy="0.0 0.0 0.0" xyz="0.0 0.0 0.5825"/>
  </joint>
  <link name="low_body_link">
    <inertial>
      <mass value="50.0"/>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <inertia ixx="5.9155184671250005" ixy="0.0" ixz="0.0" iyy="5.9155184671250005" iyz="0.0" izz="0.520833125"/>
    </inertial>
    <visual>
      <orign rpy="0.0 0.0 0.0" xyz="0.0 0.0 0.0"/>
      <geometry>
        <box size="0.25 0.25 1.165"/>
      </geometry>
    </visual>
    <collision>
      <orign rpy="0.0 0.0 0.0" xyz="0.0 0.0 0.0"/>
      <geometry>
        <box size="0.25 0.25 1.165"/>
      </geometry>
    </collision>
  </link>
  <joint name="left_panel_joint" type="fixed">
    <parent link="low_body_link"/>
    <child link="left_panel_link"/>
    <origin rpy="0.0 0.0 0.0" xyz="-0.08 0.0 0.6174999999999999"/>
  </joint>
  <link name="left_panel_link">
    <inertial>
      <mass value="5.0"/>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <inertia ixx="0.0280833221" ixy="0.0" ixz="0.0" iyy="0.0054166645000000005" iyz="0.0" izz="0.0294166549"/>
    </inertial>
    <visual>
      <orign rpy="0.0 0.0 0.0" xyz="0.0 0.0 0.0"/>
      <geometry>
        <box size="0.09 0.25 0.07"/>
      </geometry>
    </visual>
    <collision>
      <orign rpy="0.0 0.0 0.0" xyz="0.0 0.0 0.0"/>
      <geometry>
        <box size="0.09 0.25 0.07"/>
      </geometry>
    </collision>
  </link>
  <joint name="right_panel_joint" type="fixed">
    <parent link="low_body_link"/>
    <child link="right_panel_link"/>
    <origin rpy="0.0 0.0 0.0" xyz="0.08 0.0 0.6174999999999999"/>
  </joint>
  <link name="right_panel_link">
    <inertial>
      <mass value="5.0"/>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <inertia ixx="0.0280833221" ixy="0.0" ixz="0.0" iyy="0.0054166645000000005" iyz="0.0" izz="0.0294166549"/>
    </inertial>
    <visual>
      <orign rpy="0.0 0.0 0.0" xyz="0.0 0.0 0.0"/>
      <geometry>
        <box size="0.09 0.25 0.07"/>
      </geometry>
    </visual>
    <collision>
      <orign rpy="0.0 0.0 0.0" xyz="0.0 0.0 0.0"/>
      <geometry>
        <box size="0.09 0.25 0.07"/>
      </geometry>
    </collision>
  </link>
  <joint name="top_panel_joint" type="fixed">
    <parent link="low_body_link"/>
    <child link="top_panel_link"/>
    <origin rpy="0.0 0.0 0.0" xyz="0.0 0.0 0.6975"/>
  </joint>
  <link name="top_panel_link">
    <inertial>
      <mass value="10.0"/>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <inertia ixx="0.0588333098" ixy="0.0" ixz="0.0" iyy="0.0588333098" iyz="0.0" izz="0.104166625"/>
    </inertial>
    <visual>
      <orign rpy="0.0 0.0 0.0" xyz="0.0 0.0 0.0"/>
      <geometry>
        <box size="0.25 0.25 0.09"/>
      </geometry>
    </visual>
    <collision>
      <orign rpy="0.0 0.0 0.0" xyz="0.0 0.0 0.0"/>
      <geometry>
        <box size="0.25 0.25 0.09"/>
      </geometry>
    </collision>
  </link>
  <!-- colorize adapter station -->
  <gazebo reference="low_body_link">
    <material>Gazebo/Grey</material>
  </gazebo>
  <gazebo reference="left_panel_link">
    <material>Gazebo/Grey</material>
  </gazebo>
  <gazebo reference="right_panel_link">
    <material>Gazebo/Grey</material>
  </gazebo>
  <gazebo reference="top_panel_link">
    <material>Gazebo/Grey</material>
  </gazebo>
</robot>

